build-system/compile/bundles.config.extensions.json

```diff
@@ -94,6 +94,12 @@
     "options": {"hasCss": true}
   },
   {"name": "amp-base-carousel", "version": "1.0", "latestVersion": "0.1"},
+  {
+    "name": "amp-bento",
```

Comment:
suggestion: Can this just be `bento` or `bento-amp` since we don't need the CE name?